file-formats/evto/examples/z_aff_example.evto.json

```diff
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example Event Object.",
+    "originalLanguage": "en"
+  },
+  "majorVersion": 1,
+  "minorVersion": 0,
+  "patchVersion": 4,
+  "sapObjectType": "Z_AFF_EXAMPLE"
```

Comment:
I am not sure: Does it make sense to move the SAP object type at the first location?

Reply:
Yes, I will incorporate this suggestion.